Makefile

```diff
@@ -1,5 +1,25 @@
 .DEFAULT_GOAL := default
 
+include config.mk
+
+SERVICES := auth batch ci memory notebook monitoring website
+SERVICES_IMAGES := $(patsubst %, %-image, $(SERVICES))
+SERVICES_MODULES := $(SERVICES) gear web_common
+CHECK_SERVICES_MODULES := $(patsubst %, check-%, $(SERVICES_MODULES))
+
+HAILTOP_VERSION := hail/python/hailtop/hail_version
+SERVICES_IMAGE_DEPS = hail-ubuntu-image $(HAILTOP_VERSION) $(shell git ls-files hail/python/hailtop gear web_common)
+
+EMPTY :=
+SPACE := $(EMPTY) $(EMPTY)
+EXTRA_PYTHONPATH := hail/python:$(subst $(SPACE),:,$(SERVICES_MODULES))
+ifeq ($(PYTHONPATH), "")
+PYTHONPATH := $(EXTRA_PYTHONPATH)
+else
+PYTHONPATH := $(PYTHONPATH):$(EXTRA_PYTHONPATH)
+endif
+PYTHON := PYTHONPATH=$(PYTHONPATH) python3
```

Comment:
I think a `"$(PYTHONPATH)"` here (for bash's sake) is necessary in case someone has a space in their python path.